### PR TITLE
Pls add me :)

### PR DESCRIPTION
**Pull request by KunalGehlot**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import advaithunnikrishnan from "../../photos/advaithunnikrishnan.jpg";
 import AsishRaju from "../../photos/AsishRaju.jpg";
 import Aineshphanithi from "../../photos/Aineshphanithi.jpg";
 import chrisft25 from "../../photos/chrisft25.jpg";
+import kunalgehlot from "../../photos/kunalgehlot.jpg";
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:14]:
```suggestion
import KunalGehlot from "../../photos/kunalgehlot.jpg";
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@KunalGehlot It's case sensitive.

File: src/components/json/worldData.js (hunk 1 above)
Comment (KunalGehlot) [reply src/components/json/worldData.js:14]: Yeah sorry, I missed it

**Review by abhinavsri360 — APPROVED** [review]
Thanks for adding yourself!

**Pull request merged**